cmd/serve.go

```diff
@@ -69,18 +79,42 @@ func cmdServe(cmd *cobra.Command, args []string) {
 		case <-ctx.Done():
 			return
 		case e := <-events:
-			switch e {
+			switch e.Type {
 			case kube.NewPodEvent:
-				metrics.GlobalPodMetrics.TotalNumberPods.Add(1)
+				podlog := log.New("pod", e.PodSelfLink)
+
+				cgPath := buildCgroupPath(cgroupRoot, e.PodUID, e.PodQOSClass)
```

Comment:
I think you could follow [this](https://github.com/kubernetes/client-go/blob/7c69e980210777a6292351ac6873de083526f08e/kubernetes/typed/apps/v1beta1/deployment.go#L128-L134) and do something like `client.Get().Namespace("foo").Resource("bar").URL()`.